### ISSUE TITLE
IllegalArgumentException v1.7.10

### ISSUE THREAD

**Issue by crankycoder** [mozilla/MozStumbler#1602]
``` java
    at org.mozilla.mozstumbler.service.stumblerthread.motiondetection.MotionSensor$NetworkLocationChangeDetector.start(MotionSensor.java:175)
```

**Comment by garvankeeley** [comment]
This would be on a device with no location service. Any idea what the device specifics are?

**Comment by crankycoder** [comment]
Ach.  Sorry. I deleted the report already, but it was some Sony Xperia device.

**Comment by crankycoder** [comment]
Sony ST21i, Android rev 4.0.4
Fingerprint: Sony/ST21i_1264-4261/ST21i:4.0.4/11.0.A.4.27/46N97w:user/release-keys

**Comment by crankycoder** [comment]
Fixed in #1626
